Binary files orig/strawr/src/.Makevars.ucrt.swp and patched/strawr/src/.Makevars.ucrt.swp differ
diff -Nru orig/strawr/src/Makevars.ucrt patched/strawr/src/Makevars.ucrt
--- orig/strawr/src/Makevars.ucrt	2023-03-29 01:12:36.000000000 +0200
+++ patched/strawr/src/Makevars.ucrt	2024-02-08 14:44:13.814549000 +0100
@@ -1,2 +1,21 @@
-CRT=-ucrt
-include Makevars.win
+
+ifeq (,$(shell pkg-config --version 2>/dev/null))
+  PKG_LIBS= \
+	-lwinhttp -lcurl -lssh2 -lz -lssl -lcrypto -lgdi32 -lcrypt32 -lwldap32 \
+        -lzstd -lgcrypt -lgpg-error -lidn2 -lbcrypt -lunistring -liconv -lws2_32
+else
+  PKG_LIBS=$(shell pkg-config --libs libcurl)
+endif
+
+PKG_CPPFLAGS= \
+	-DCURL_STATICLIB -DSTRICT_R_HEADERS
+
+all: clean winlibs
+
+clean:
+	rm -f $(SHLIB) $(OBJECTS)
+
+winlibs: clean
+	echo '#include <curl/curl.h>' | $(CC) $(PKG_CPPFLAGS) -std=gnu99 -E -xc - | grep "^[ \t]*CURLOPT_.*," | sed s/,// > ../tools/option_table.txt
+
+.PHONY: all winlibs clean
